onnxruntime/core/optimizer/relu_clip_fusion.cc

```diff
@@ -12,7 +12,7 @@ Status FuseReluClip::Apply(Graph& graph, Node& node, RewriteRuleEffect& rule_eff
   // get the following Clip node before we delete the Relu node
```

Comment:
Make this a rewrite rule?

Reply:
It could be at some point, but that should be a separate PR.